### PR TITLE
Update Windows and RBAC sections in Logging

### PR DESCRIPTION
**Pull request by nickgerace**
Update Windows section to remove fluentd configcheck warning. Refactor
RBAC section to be more streamlined. Add syslog note. Perform general edits on grammar
and style.

Relevant issue: https://github.com/rancher/docs/issues/2867

### CODE REVIEW

File: content/rancher/v2.x/en/logging/v2.5/_index.md [hunk 1]
```diff
@@ -177,7 +176,7 @@ spec:
     - "devteam-splunk"
 ```
 
-For the final example we create an output to write logs to a destination that is not supported out of the box (e.g. syslog):
+**Unsupported Ouput:** For the final example, we create an output to write logs to a destination that is not supported out of the box (e.g. syslog):
```
Comment (paynejacob) [inline content/rancher/v2.x/en/logging/v2.5/_index.md:180]: now that we support syslog we should update this example to use the next most popular unsupported output.

Comment (paynejacob) [reply content/rancher/v2.x/en/logging/v2.5/_index.md:180]: That could be a different ticket though.

**Review by paynejacob — APPROVED** [review]

File: content/rancher/v2.x/en/logging/v2.5/_index.md (hunk 1 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/logging/v2.5/_index.md:180]: That would probably be best. I believe this example uses a different syslog plugin, so it *should* be fine for the immediate future. I do agree though that we should evaluate the next most popular one for a future PR/issue.

File: content/rancher/v2.x/en/logging/v2.5/_index.md [hunk 2]
```diff
@@ -11,19 +11,17 @@ weight: 1
...
 
-
 # Changes in Rancher v2.5
 
 The following changes were introduced to logging in Rancher v2.5:
 
-- The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers Rancher's logging in place of the former, in-house logging solution.
-- [Fluent Bit](https://fluentbit.io/) is now used to aggregate the logs. [Fluentd](https://www.fluentd.org/) is used for filtering the messages and routing them to the outputs. Previously, only Fluentd was used.
-- Logging can be configured with a Kubernetes manifest, because now the logging uses a Kubernetes operator with Custom Resource Definitions.
+- The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers Rancher's logging solution in place of the former, in-house one.
```
Comment (cbron) [inline content/rancher/v2.x/en/logging/v2.5/_index.md:18]:
```suggestion
- The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers Rancher's logging solution in place of the former, in-house solution.
```

File: content/rancher/v2.x/en/logging/v2.5/_index.md [hunk 3]
```diff
@@ -297,19 +298,17 @@ spec:
...
+In the above example, we ensure that our pod only runs on Linux nodes, and we add a `toleration` for the taint we have on all of our Linux nodes.
 You can do the same with Rancher's existing taints, or with your own custom ones.
 
 ### Windows Support
 
-Clusters with Windows worker support logging with some small caveats:
-
-1. Windows node logs are currently unable to be exported.
-2. ```fluentd-configcheck``` pod(s) will fail due to an [upstream issue](https://github.com/banzaicloud/logging-operator/issues/592), where ```tolerations``` and ```nodeSelector``` settings are not inherited from the ```logging-operator```.
+Clusters with Windows workers support logging, but Windows node logs are currently unable to be exported.
```
Comment (cbron) [inline content/rancher/v2.x/en/logging/v2.5/_index.md:306]:
```suggestion
Clusters with Windows workers support exporting logs from Linux nodes, but Windows node logs are currently unable to be exported.
```

File: content/rancher/v2.x/en/logging/v2.5/_index.md [hunk 4]
```diff
@@ -265,29 +264,31 @@ spec:
...
+By default, Rancher taints all Linux nodes with `cattle.io/os=linux`, and does not taint Windows nodes.
+The logging stack pods have `tolerations` for this taint, which enables them to run on Linux nodes.
+Moreover, we can populate the `nodeSelector` to ensure that our pods *only* run on Linux nodes.
 Let's look at an example pod YAML file with these settings...
 
 ```yaml
 apiVersion: v1
 kind: Pod
-# metadata:
+# insert metadata
```
Comment (cbron) [inline content/rancher/v2.x/en/logging/v2.5/_index.md:289]: nit: 
```suggestion
# metadata...
```
same for containers, thats the way I've seen it done in other documentation. We should match whatever K8s docs do.

**Review by cbron — COMMENTED** [review]
just minor things

File: content/rancher/v2.x/en/logging/v2.5/_index.md (hunk 2 above)
Comment (nickgerace) [reply content/rancher/v2.x/en/logging/v2.5/_index.md]: done

**Review by catherineluse — APPROVED** [review]

**Pull request merged**